django-web-app/blog/urls.py

```diff
@@ -1,6 +1,20 @@
 from django.urls import path
+from . views import (
+    PostListView, 
+    PostDetailView, 
+    PostCreateView,
+    PostUpdateView,
+    PostDeleteView
+)
 from . import views
+
 urlpatterns = [
-    path('', views.home, name='blog-home'),
+    path('', PostListView.as_view(), name='blog-home'),
+    path('post/<int:pk/>', PostDetailView.as_view(), name='post-detail'),
```

Comment:
cause of the problem here as you can see on the error logs too.
Syntax error: do ```post/<int:pk>/``` instead of ```post/<int:pk/>``` .

Reply:
```<int:pk>``` is a phrase by itself, telling that you are expecting an integer pk.